src/tinypeel/Peeling/PeelingIO.py

```diff
@@ -189,7 +189,7 @@ def doubleIfNotMissing(calledGenotypes):
 def setMissing(calledGenotypes, matrix, thresh):
     nLoci = len(calledGenotypes)
     for i in range(nLoci):
-        if matrix[calledGenotypes[i], i] < thresh:
+        if matrix[calledGenotypes[i], i] <= thresh:
```

Comment:
@XingerTang let's have this change in the right commit or then changes become all mixed up

Reply:
@XingerTang I am loosing track in this PR - have you reverted back this change (the original was <, changed to <=, but this shuold be done in another PR focused on that issue)? We really should strive doing one thing at a time or we will be completely lost what change was made where;)

Reply:
@XingerTang ok, I get it now. Consider this resolved.